install/mac/Makefile

```diff
@@ -20,8 +20,8 @@ export-certificates:
 install: prereq install-software export-certificates vagrant-certificates
 
 install-software:
-	brew cask install virtualbox
-	brew cask install vagrant
+	brew install --cask virtualbox
+	brew install --cask vagrant
```

Comment:
Doesn't this actually change to using `brew` instead of `brew cask`? I've understood it as `brew cask` being an extension of Homebrew.

Anyways, if it works I'm OK with it :+1:

Reply:
Yup - deprecated https://brew.sh/2020/12/01/homebrew-2.6.0/